docs/index.rst

```diff
@@ -0,0 +1,46 @@
+p4c is a reference compiler for the P4 programming language. It supports both P4-14 and P4-16; you can find more information about P4 here and the specifications for both versions of the language here. One fact attesting to the level of quality and completeness of p4c's code is that its front-end code, mid-end code, and p4c-graphs back end are used as the basis for at least one commercially supported P4 compiler.
```

Comment:
I believe this was committed accidentally.

Reply:
Yes and no, I am currently working to update the documentation for the p4 compiler site. So far I was able to create a roadmap of how I would be able to create a documentation website for p4c following this https://tvm.apache.org/docs site in mind. That file was just part of the big change.

Reply:
We should add this change in a separate PR. It is good to keep your pull requests small and targeted.

Reply:
Great sounds good to me. I would then create a separate PR and commit there.

Reply:
Please do so. Until then I can not approve this PR.